index.html

```diff
@@ -0,0 +1,200 @@
+                    <p class="card-info">Building Resilient Supply Chains in agriculture including strategies for mitigating risks.</p>
+                </div>
+                <div class="program-card">
+                    <img class="card-image" src="images/sun-icon.png" alt="">
+                    <h3 class="card-title main-color">Sustainable Agriculture</h3>
+                    <p class="card-info">Balancing Profitability and Environmental Stewardship as to being a good climate advocate</p>
+                </div>
+                <div class="program-card">
+                    <img class="card-image" src="images/organic-food-icon.png" alt="">
+                    <h3 class="card-title main-color">AE Workshop</h3>
+                    <p class="card-info">Meet with large farm owners, network, disscuss and exchanges ideas and troubleshoot problems</p>
+                </div>
+            </div>
+            <div class="button-space">
+                <button class="large-button"><a class="large-button-link" href="http://">Join AE Global Summit 2023 </a></button>
+            </div>
+            <div class="desk-prog-info">
+                <p>SEE THE WHOLE PROGRAM</p>
+            </div>
+        </div>
+    </section>
+    <section class="third-section other-parts">
+        <div class="section-content about-content">
+            <h3 class="title-text speaker-title">Featured Speakers<hr class="horizontal-line"></h3>
+            <div class="desk-speaks">
+                <div class="row-1">
+                   <div class="desk-card">
+                        <img class="desk-card-img" src="images/speaker-1.jpg" alt="speaker-picture">
+                        <div class="desk-card-content">
+                            <h5 class="desk-speaker-name">Samuel Benard</h5>
+                            <p class="desk-speaker-qualification">Risk analyst at International Fund for Agricultural Development</p>
+                            <hr class="speaker-hr">
+                            <p class="desk-small-text">He worked with farmers and agricultural businesses to improve their risk management practices, providing training and consultation.</p>
+                        </div>
+                   </div>
+                   <div class="desk-card">
+                        <img class="desk-card-img" src="images/speaker-4.jpg" alt="speaker-picture">
+                        <div class="desk-card-content">
+                            <h5 class="desk-speaker-name">Akintunde Babalola</h5>
+                            <p class="desk-speaker-qualification">Ph.D. in Horticulture with expertise in urban agriculture and sustainable food systems.</p>
+                            <hr class="speaker-hr">
+                            <p class="desk-small-text">Akin published several articles on horticultural science in top-tier academic journals, which have been widely cited and contributed to advances in the field.</p>
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly ensure that the speakers' section is generated dynamically with Javascript.